hasura/helm/hasura/templates/postgres.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.hasura.postgresql.host "plural-hasura" }}
```

Comment:
Maybe it would be better to use another key for this. Something like:

```yaml
hasura:
  pgClient:
    external:
      enabled: true
      operatorEnabled: true
```